### PR TITLE
feat: add a tool for generating packages

### LINKED ISSUE #105: Add a tool for generating packages
## Is your feature request related to a problem? Please describe.

The number of packages available for APM is limited.

## Describe the solution you'd like

Provide a tool to create the XML for the package to accelerate the addition of data.

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Add a tool to APM to create the XML of the package.

## Related Issue

Close #105 

## How Has This Been Tested?

I tested the functionality using the following download URL.
```
https://github.com/oov/aviutl_psdtoolkit/releases
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/package_maker_preload.js [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+  '.tiff',
+  '.tif',
+  '.webp',
+  '.svg',
+];
+const searchFiles = (dirName, directory = false, ignoreExtention = []) => {
+  const dirents = fs.readdirSync(dirName, {
+    withFileTypes: true,
+  });
+  console.log(ignoreExtention);
```
Comment (hal-shu-sato) [inline src/package_maker_preload.js:24]: Is this log for the test?

File: src/package_maker.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  width: 100%;
+  height: 100%;
+}
+body {
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+h1 {
+  margin: 0;
+}
```
Comment (hal-shu-sato) [inline src/package_maker.css:13]: Are they used?
It doesn't seem to work.

**Review by hal-shu-sato — COMMENTED** [review]
I have some questions.
But what an amazing feature it is!!!

**Comment by mitosagi** [comment]
I fixed them.

**Review by hal-shu-sato — APPROVED** [review]
Thank you for fixing it.

**Pull request merged**